### ISSUE TITLE
IOS and Androed onCurrentPageChanged (Number pagenumber) Not fireing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#399]
```
What steps will reproduce the problem?
1.  View a document on IOS or Andriod
2.  Monitor for onCurrentPageChanged event 
3.  Does not fire!

What is the expected output? 

To see the event on CurrentPageChanged fire...


What do you see instead?

Nothing... 


What version of the product are you using? 

2.2.1 FlexPaper_2.2.1_c_customizable

On what operating system?

Debian

Please provide any additional information below.

```

Original issue reported on code.google.com by `andrew.s...@gmail.com` on 20 Mar 2014 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
fixed in our upcoming 2.2.4 build. Thanks for reporting! 
```

Original comment by `erik.eng...@gmail.com` on 23 Mar 2014 at 11:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hello, 
On iOS, this fixes the problem of not firing event but introduced a new small 
bug:

$(window).on('onCurrentPageChanged', function (e, page_number) {
    console.log(page_number);
});
For example: the PDF has 2 pages, if you click previous button on page 1, you 
can't see any new page but the console.log above will log page_number is 0. 
Similarly, if you click next button on page 2, it will log 3. 
Expected result: page_number should not go out of range 1-2.
```

Original comment by `andrew.s...@gmail.com` on 24 Mar 2014 at 9:45
